src/data/models.yaml

```diff
@@ -181,3 +181,8 @@ entries:
     homepage: https://cmungall.github.io/mianct-schema/
     github_repo: cmungall/mianct-schema
     schema_relative_path: model/schema/mianct.yaml
+  iSamples:
+    homepage: https://github.com/isamplesorg/metadata
```

Comment:
this is fine but did you know you can easily make a homepage that goes here: https://isamplesorg.github.io/metadata?

Reply:
Thank you so much! I am not sure about it. Let me ask my PI.